conctr.agent.class.nut

```diff
@@ -10,8 +10,13 @@ class Conctr {
     static version = [1,0,0];
 
     static DATA_EVENT = "conctr_data";
+    static LOCATION_REQ = "conctr_get_location";
+    static LOCATION_RESP = "conctr_location";
+    static AGENT_OPTS = "conctr_agent_options";
+    static SOURCE_DEVICE = "impdevice";
```

Comment:
I don't understand that response. I'd like to see the platform knowing if the source was the agent or the device. Once you have that data you might as well use it in whatever other checks you have.